src/Kunstmaan/NodeSearchBundle/EventListener/NodeIndexUpdateEventListener.php

```diff
@@ -44,7 +44,9 @@ public function onPostPersist(NodeEvent $event)
     private function index(NodeEvent $event)
     {
         $nodeSearchConfiguration = $this->container->get('kunstmaan_node_search.search_configuration.node');
-        $nodeSearchConfiguration->indexNodeTranslation($event->getNodeTranslation(), true);
+        $nodeTranslation = $event->getNodeTranslation();
+        $nodeSearchConfiguration->indexNodeTranslation($nodeTranslation, true);
+        $nodeSearchConfiguration->indexChildren($event->getNode(), $nodeTranslation->getLang());
```

Comment:
Can we please think about abstracting it into some kind of `ChildrenNeedReindexingDecisionMakerInterface`? I have nodes that index elements of their parents, so this kind of behaviour would apply there.